jib-cli/README.md

```diff
@@ -181,7 +191,7 @@ layers:
         - dest: "/images"            
 ```
 
-#### Layers behavior
+#### Layers Behavior
```

Comment:
I am actually considering moving the "fully annotated jib.yaml" and this section to the bottom of the page in a new "reference" section while adding links. I think it's important to quickly give a short and concise "getting started" impression, so that people may just be tempted to try out. If they become really interested, they can consult the detailed references. We don't want this tool to look complex to use. Your thoughts?

Reply:
Yeah, I like the idea of having a reference section for this. Having all this information in the beginning can be a little overwhelming.